cjs/interface/document.js

```diff
@@ -159,6 +159,13 @@ class Document extends NonElementParentNode {
 
   get isConnected() { return true; }
 
+  /**
+   * @protected
+   */
+   _getParent() {
+    return this[EVENT_TARGET];
```

Comment:
returning `this.defaultView` wasn't working as expected and since this is used only for bubbling that was updated to work in that context.